### ISSUE TITLE
_cleantarget_

### ISSUE THREAD

**Issue by codingo** [codingo/Interlace#64]
I'm unsure if this will be the final variable name but as stated in #63, slashes do break the piping of the file.

Since targets could take the following forms:

```
https://example.com/
https://example.com
https://example.com/dashboard
https://example.com/dashboard/
```

The proposal is to have a variable `_cleantarget_`. This will print the contents of `_target_` but will remove `https://`, `http://`, any trailing `/` and then replace any remaining `/` with `-`.  This should allow this to be used to generate output filenames, even when URL's to specific locations are added.

This should help to close out #63 as well as aid in cases such as #54.

**Comment by codingo** [comment]
I think we should also add a `_cleanproxy_` with the same logic in case people want to store the used proxy for the request in their output (rare edge case, but easy to implement).

**Comment by iamOgunyinka** [comment]
Needs more clarification for how this should work. It seem _cleanproxy_ is the intended name to use but implementation require more technical specification.

**Comment by prodigysml** [comment]
It's a small change. Basically taking each var and doing `target.replace("/", "-")`, then giving a separate variable name. I should be able to patch this now.

**Comment by prodigysml** [comment]
Happy for you to take the issue if you want @iamOgunyinka

**Comment by iamOgunyinka** [comment]
No, go ahead. You seem to understand the problem better anyway.

**Comment by iamOgunyinka** [comment]
@prodigysml has #54 been fixed yet? I've got some free time now and could maybe implement it in a few.

**Comment by prodigysml** [comment]
I think it has, but feel free to test and verify. Might've missed something

**Comment by codingo** [comment]
#73 being tested for this now

**Comment by codingo** [comment]
Closed with #73
